### PR TITLE
[#384] Add tvOS support

### LINKED ISSUE #384: tvOS support
Add a new target to the SDK to build the framework for tvOS.

The framework should still be named "Buy".

### PR DESCRIPTION
**Pull request by EishanVijay**
## What
Adds a new target to the SDK to support tvOS

Resolves #384 

@bgulanowski @davidmuzi @gabrieloc @dbart01

### CODE REVIEW

File: Mobile Buy SDK/BuyTV/Buy.h [hunk 1]
```diff
@@ -0,0 +1,90 @@
+//
```
Comment (davidmuzi) [inline Mobile Buy SDK/BuyTV/Buy.h]: This file is ~90% the same as for iOS (minus the ApplePay and Web checkout support).  We can use the existing umbrella header and wrap the unwanted headers in `#ifndef TARGET_OS_TV`

File: Mobile Buy SDK/BuyTV/Info.plist [hunk 2]
```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (davidmuzi) [inline Mobile Buy SDK/BuyTV/Info.plist]: this is identical as well.  You can specify the existing plist from Build Settings->Info Plist File

File: Mobile Buy SDK/BuyTV/Buy.h (hunk 1 above)
Comment (EishanVijay) [inline Mobile Buy SDK/BuyTV/Buy.h]: I don't think we can use the same umbrella header for two targets

File: Mobile Buy SDK/Mobile Buy SDK/Buy.h [hunk 3]
```diff
@@ -26,6 +26,13 @@
 
 #import <UIKit/UIKit.h>
 
+#if TARGET_OS_IOS
```
Comment (davidmuzi) [inline Mobile Buy SDK/Mobile Buy SDK/Buy.h]: Should be under // Checkout support

**Comment by EishanVijay** [comment]
@davidmuzi Okay to merge?

**Pull request merged**